WeakAuras/Prototypes.lua

```diff
@@ -1024,6 +1024,7 @@ function WeakAuras.IsPlayerSpellOrOverridesAndBaseIsPlayerSpell(spell)
 end
 
 function WeakAuras.IsSpellKnownForLoad(spell, exact)
+  spell = tonumber(spell) or spell
```

Comment:
I think we should revert the breaking change, looking at it again it looks to be wrong.

Reply:
done